mkdocs.yml

```diff
@@ -10,6 +10,8 @@ theme:
     - tabs
   logo: img/logos/EESSI.png
   favicon: img/logos/EESSI.png
+  feature:
+    tabs: false
```

Comment:
just remove the `- tabs` on line 10 instead?

Reply:
Yup, fixing now